src/submission/schema/schema-api.ts

```diff
@@ -93,10 +93,11 @@ export const getTemplate = async (req: Request, res: Response) => {
   }
 
   const template = createTemplate(schema);
+
   return res
     .status(200)
     .contentType('text/tab-separated-values')
-    .attachment(`${schemaName}.tsv`)
+    .attachment(`${schemaName}_dictionary_v${schemasDictionary.version}.tsv`)
```

Comment:
changes in this file are for this ticket: https://github.com/icgc-argo/argo-clinical/issues/363, it was small so I snuck it in here